lib/models/project.js

```diff
@@ -530,6 +530,32 @@ Project.prototype.findAddonByName = function(name) {
 };
 
 /**
+  Generate test file contents.
+
+  This method is supposed to be overwritten by test framework addons
+  like `ember-cli-qunit` and `ember-cli-mocha`.
+
+  @protected
```

Comment:
just noticed that `protected` is obviously bullshit here because other addons would need to use this so it should be a `public` API